### PR TITLE
feat: most simple indexer

### PR DESCRIPTION
**Pull request by florian-hoenicke**

### CODE REVIEW

File: now/hub/most_simple_indexer/most_simple_indexer.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from jina import Executor, requests
+
+
+class MostSimpleIndexer(Executor):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.index = DocumentArray()
+
+    @requests
+    def index(self, docs: Optional[DocumentArray], **kwargs):
```
Comment (makram93) [inline now/hub/most_simple_indexer/most_simple_indexer.py:13]: `docs` are no longer optional
```suggestion
    def index(self, docs: DocumentArray, **kwargs):
```

File: now/hub/most_simple_indexer/most_simple_indexer.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        self.index.extend(docs)
+
+    @requests(on='/search')
+    def search(self, docs: Optional[DocumentArray], **kwargs):
```
Comment (makram93) [inline now/hub/most_simple_indexer/most_simple_indexer.py:17]: same here

File: now/hub/most_simple_indexer/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+--find-links https://download.pytorch.org/whl/torch_stable.html
+torch==1.10.2+cpu
```
Comment (makram93) [inline now/hub/most_simple_indexer/requirements.txt:2]: Why we need pytorch here? For `match` call?

**Review by makram93 — COMMENTED** [review]
Minor comments

File: now/hub/most_simple_indexer/most_simple_indexer.py (hunk 2 above)
Comment (florian-hoenicke) [reply now/hub/most_simple_indexer/most_simple_indexer.py:17]: thanks - will change

File: now/hub/most_simple_indexer/requirements.txt (hunk 3 above)
Comment (florian-hoenicke) [reply now/hub/most_simple_indexer/requirements.txt:2]: good observation - it was a copy-paste error. I will change it

**Comment by florian-hoenicke** [comment]
@makram93 Thanks for the feedback. I implemented the required changes

**Review by makram93 — APPROVED** [review]

**Pull request merged**